shift-react-app/src/shiftReactApp.ts

```diff
@@ -0,0 +1,23 @@
+// tslint:disable:no-console
+
+import {
+  setupProxy,
+  installPackages,
+  ignoreShift,
+} from './steps';
+
+async function shiftReactApp() {
+  try {
+    let msg = setupProxy();
```

Comment:
I'd add a first step call `sanityCheck` instead of having `setupProxy` verify this so we could reorder the steps in the future.

Reply:
```typescript
const steps = [
  sanityCheck, // Must run first
  setupProxy,
  installPackages,
  ignoreShift,
];
for (const step of steps) {
  console.log(await step());
}
```